rmgpy/molecule/fragment_utils.py

```diff
@@ -310,3 +313,321 @@ def merge_frag_list(to_be_merged):
+    nearby_pi_bond_bool = any(x>1 for x in nearby_bond_orders)
+    if nearby_radical_bool ==False and nearby_pi_bond_bool == False and nearby_cl_bool ==False:
+        return True
+    else:
+        return False
+
+def return_cuttable_bonds(frag):
+    frag, single_cc_bonds = get_single_cc_bonds(frag)
+    
+    cuttable_bonds = []
+    for bond_idx_tuple in single_cc_bonds:
+        cuttable_bool = check_if_bond_is_cuttable(frag, bond_idx_tuple)
+        if cuttable_bool:
+            cuttable_bonds.append(bond_idx_tuple)
+    cuttable_bonds = list(set(cuttable_bonds))
+    return cuttable_bonds
+
+def process_new_fragment(species_smiles, starting_fragment_smiles,species_cutting_threshold = 20):
+
+    radical_result = check_if_radical_near_cutting_label(species_smiles)
+    
+
+    if radical_result != False:
+        species_fragment, nearest_cutting_label = radical_result
+        options = add_starting_fragment_cut_if_needed(species_fragment, nearest_cutting_label, starting_fragment_smiles,species_cutting_threshold = species_cutting_threshold)
+        return options
+    else:
+        pibond_result = check_if_pibond_near_cutting_label(species_smiles)
+        if pibond_result!=False:
+            species_fragment, nearest_cutting_label = pibond_result
+            options = add_starting_fragment_cut_if_needed(species_fragment, nearest_cutting_label, starting_fragment_smiles,species_cutting_threshold = species_cutting_threshold)
+            return options
+        else:
+            return species_smiles
+
+def make_new_reaction_string(species_smiles, starting_fragment_smiles, frag_list):
+    frag_list_str = " + ".join(frag_list)
+    return f"{species_smiles} + {starting_fragment_smiles} => {frag_list_str}"
+
+
+def generate_add_partial_reattachment_reactions(seed_dir, starting_fragments):
+    seed_reactions_filename = os.path.join(seed_dir, "reactions.py")
+    seed_dictionary_filename = os.path.join(seed_dir, "dictionary.txt")
+
+    with open(seed_dictionary_filename,'r') as f:
+        dictionary_lines = f.readlines()
+
+    fragment_to_name_dictionary = {}
+
+    dlines_iter = iter(dictionary_lines)
+    name = next(dlines_iter,"end").strip().strip('\n')
+    line = next(dlines_iter,"end")
+    while line != "end" and name !="end":
+        species_adjlist =""
+        line = next(dlines_iter,"end")
+        while line.strip().strip('\n') != "":
+            species_adjlist +=line
+            line = next(dlines_iter, "end")
+
+        f = Fragment().from_adjacency_list(species_adjlist)
+        fragment_to_name_dictionary[f] = name
+        name = next(dlines_iter,"end").strip().strip('\n')
+
+    rxn_strs = []
+    for f in fragment_to_name_dictionary.keys():
+        smiles = f.smiles
+        try:
+            for starting_fragment in starting_fragments:
+                options = process_new_fragment(smiles, starting_fragment,species_cutting_threshold=12)
```

Comment:
```suggestion
                options = process_new_fragment(smiles, starting_fragment, species_cutting_threshold=12)
```

Why do we use a different threshold here?